jobserver/api/releases.py

```diff
@@ -67,17 +66,7 @@ def put(self, request, workspace_name, release_hash):
         )
 
         response = Response(status=201 if created else 303)
-        response["Location"] = request.build_absolute_uri(
-            reverse(
-                "workspace-release",
-                kwargs={
-                    "org_slug": workspace.project.org.slug,
-                    "project_slug": workspace.project.slug,
-                    "workspace_slug": workspace.name,
-                    "release": release.id,
-                },
-            )
-        )
+        response["Location"] = request.build_absolute_uri(release.get_absolute_url())
```

Comment:
I wonder if we should actually link them to the Workspace url here, rather than the Release?